src/kakarot/instructions/block_information.cairo

```diff
@@ -103,76 +106,49 @@ namespace BlockInformation {
         return evm;
 
         basefee:
-        // Since Kakarot does not implement EIP1559,
-        // there is no priority fee, therefore gasPrice == baseFeePerGas
-        // TODO: since gas_price is a felt, it might panic when being cast to a Uint256.low,
-        // Add check gas_price < 2 ** 128 at creation of EVM
-        // `split_felt` might be too expensive for this if we know gas_price < 2 ** 128
         let evm = cast([fp - 3], model.EVM*);
-        tempvar gas_price = evm.message.gas_price;
-        tempvar result = Uint256(gas_price, 0);
-        jmp end_constant;
-
-        end_constant:
-        // Rebind unused args with fp
-        let syscall_ptr = cast([fp - 10], felt*);
-        let pedersen_ptr = cast([fp - 9], HashBuiltin*);
-        let range_check_ptr = [fp - 8];
-        let bitwise_ptr = cast([fp - 7], BitwiseBuiltin*);
         let stack = cast([fp - 6], model.Stack*);
-        let memory = cast([fp - 5], model.Memory*);
-        let state = cast([fp - 4], model.State*);
-        let evm = cast([fp - 3], model.EVM*);
+        Stack.push_uint128(evm.message.env.gas_price);
```

Comment:
should be `base_fee_per_gas`

Reply:
same than in KKT1 and before, see also https://github.com/kkrt-labs/kakarot-ssj/blob/7ebb868f292816b7f94d1a2b35474d463715cda1/crates/evm/src/instructions/block_information.cairo#L133C13-L133C13

will be the case until we implement EIP 1559